crates/provider/src/kubernetes/client.rs

```diff
@@ -145,7 +148,7 @@ impl KubernetesClient {
             .await
             .map_err(|err| Error::from(anyhow!("error while creating pod {name}: {err}")))?;
 
-        await_condition(pods, name, conditions::is_pod_running())
+        await_condition(pods, name, helpers::is_pod_ready())
```

Comment:
Why this modification ? ```is_pod_running()``` doesn't works ?

Reply:
If the pod have some `probing` (like the one I added to the file server) k8s reports running and then (ones is verified) the `Ready` state. I think checking `ready` give us the possibility to use the built-in probes machinery. For pods not setting any probes the ready event is set at the same time that the running.